recipes/qpdf/all/conanfile.py

```diff
@@ -0,0 +1,168 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.files import get, copy, rmdir
+from conan.tools.build import check_min_cppstd
+from conan.tools.scm import Version
+from conan.tools.files import patch, export_conandata_patches, get, copy, rmdir
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+from conan.tools.microsoft import is_msvc, check_min_vs
+from conan.tools.env import VirtualBuildEnv
+import os
+
+required_conan_version = ">=1.52.0"
+
+class PackageConan(ConanFile):
+    name = "qpdf"
+    description = "QPDF is a command-line tool and C++ library that performs content-preserving transformations on PDF files."
+    license = "Apache-2.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/qpdf/qpdf"
+    topics = ("pdf")
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "with_crypto": ["native", "openssl", "gnutls"],
```

Comment:
never head of this project before but what is "native" here? there's no requirements added

Reply:
It is just going for specific cmake options, telling qpdf to use its own crypto internal provider. Is there some existing working for such a selection? I though about native or internal. I just didn't wont to go for False here, since crypto is still in place, but directly shipped with the qpdf project itself.

Reply:
I will go for "internal" instead. I think it is slightly more descriptive.